unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -743,6 +763,18 @@ private void UnEquipWearable(string wearableId, UnequipWearableSource source, bo
 
             if (updateAvatarPreview)
                 UpdateAvatarModel(model.ToAvatarModel());
+
+            if (wearable.data.blockVrmExport)
+            {
+                vrmBlockingWearablesList.Remove(shortenedWearableId);
+                CheckVRMExportRequirements();
+            }
+        }
+
+        private void CheckVRMExportRequirements()
```

Comment:
Perhaps a rename: UpdateVRMExportWarning

Reply:
Thanks, I like it!